packages/flutter/lib/src/material/data_table.dart

```diff
@@ -1234,11 +1234,14 @@ class DataTable extends StatelessWidget {
         type: MaterialType.transparency,
         borderRadius: border?.borderRadius,
         clipBehavior: clipBehavior,
-        child: Table(
-          columnWidths: tableColumns.asMap(),
-          defaultVerticalAlignment: TableCellVerticalAlignment.middle,
-          children: tableRows,
-          border: border,
+        child: Semantics(
+          role: SemanticsRole.table,
```

Comment:
Should this be built in to the `Table` widget instead?

Reply:
updated, thank you!